pkg/odo/cli/url/create.go

```diff
@@ -123,6 +131,16 @@ func (o *URLCreateOptions) Complete(_ string, cmd *cobra.Command, args []string)
 			} else {
 				o.urlType = envinfo.ROUTE
 			}
+			if len(o.path) > 0 && (strings.HasPrefix(o.path, "/") || strings.HasPrefix(o.path, "\\")) {
```

Comment:
Errrrrr... is there a better way to check for this? I think you should be using https://golang.org/pkg/path/filepath/#FromSlash so you can have a uniform return regardless of front or backslashes.

Reply:
I think  `FromSlash` only checks for `/` and replace with the current operating system's file separator. This check here is to check and remove if there is a leading front or backslashes, in order to add `/` in front of the path in general. As a user can enter either `pathhewants` or `/pathhewants` as the path during url creation.